### PR TITLE
feat(uptime-kuma): use PUID/PGID & protect with authelia by default

### PR DESCRIPTION
**Pull request by JigSawFr**
# How Has This Been Tested?

- [x] App secured by Authelia SSO authentication system.
- [x] Custom UID/GID is set to the container to run under Saltbox user.

### CODE REVIEW

File: roles/uptime_kuma/tasks/main.yml [hunk 1]
```diff
@@ -23,5 +23,13 @@
 - name: Create directories
   ansible.builtin.include_tasks: "{{ resources_tasks_path }}/directories/create_directories.yml"
 
+- name: "Migrator | Reset ownership of 'opt' directory for this app"
```
Comment (owine) [inline roles/uptime_kuma/tasks/main.yml:26]: The directory creation resource called above takes care of this. If it needs to recurse there is an additional var we can define, but it’s not recursing in your migrator.

**Review by owine — CHANGES_REQUESTED** [review]

File: roles/uptime_kuma/tasks/main.yml (hunk 1 above)
Comment (JigSawFr) [reply roles/uptime_kuma/tasks/main.yml:26]: @owine does it take care at each re-run of roles ? or just at initial dir creation ?

Comment (JigSawFr) [reply roles/uptime_kuma/tasks/main.yml:26]: It works as said, thanks ! :)

**Review by maximuskowalski — APPROVED** [review]

**Comment by owine** [comment]
Are we sure we don’t need to reset permissions recursively? If an existing setup has files owned by root it would likely break.

**Pull request merged**